microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/properties/PropertiesFileCompletions.java

```diff
@@ -229,6 +229,19 @@ private static void collectPropertyKeySuggestions(int offset, Node node, Propert
 				insertText.append(' ');
 			}
 
+			// replace with current property value if exist
+			if (node != null && node.getNodeType() == NodeType.PROPERTY_KEY) {
+				String propertyValue = ((PropertyKey) node).getProperty().getPropertyValue();
+				if (propertyValue != null && propertyValue.isEmpty() == false) {

```

Comment:
for a method that returns a boolean, it's common to use the `!` operator to check for `false` (ie. the negation is true).

Reply:
Also, I'm wondering what if the property value is in fact empty. Should we not preserve it anyways ? It might be intentionally empty.